internal/ethapi/api.go

```diff
@@ -2021,3 +2066,12 @@ func toHexSlice(b [][]byte) []string {
 	}
 	return r
 }
+
+// ParseHexOrString tries to hexdecode b, but if the prefix is missing, it instead just returns the raw bytes
+func parseHexOrString(str string) ([]byte, error) {
```

Comment:
Nitpick, perhaps we can move this function into common package, but feel free to ignore this comment.